Tools/plugins/UnpackedCandidateTracks.cc

```diff
@@ -89,70 +116,254 @@ void JMTUnpackedCandidateTracks::produce(edm::Event& event, const edm::EventSetu
   edm::Handle<pat::PackedCandidateCollection> packed_candidates;
   event.getByToken(packed_candidates_token, packed_candidates);
 
+  edm::Handle<pat::PackedCandidateCollection> lost_candidates;
+  event.getByToken(lost_candidates_token, lost_candidates);
+
+  edm::Handle<pat::MuonCollection> muons;
+  event.getByToken(muons_token, muons);
+
+  edm::Handle<pat::ElectronCollection> electrons;
+  event.getByToken(electrons_token, electrons);
+
   auto tracks = std::make_unique<reco::TrackCollection>();
-  auto lost_tracks = std::make_unique<reco::TrackCollection>();
```

Comment:
@abbywarden have we dropped lost tracks entirely? I suspect we don't want to do that for generic tracks. I see the lost_candidates up above, but I think this is something different, maybe for lost tracks related to leptons? I could be getting confused though.

Reply:
Ah I should completely remove lost_candidates. lost_tracks was a collection I had momentarily when I was looking into using both packed_candidates and lost_candidates to find tracks. I was just keeping lost_tracks separate. But deleted lost_tracks when I saw it would not be beneficial to adding them in.